app/models/strategyModel.js

```diff
@@ -53,5 +53,8 @@ const StartegySchema = new mongoose.Schema(
   },
 );
 
+StartegySchema.statics.INVESTMENT_TYPE_REGULAR = '1';
```

Comment:
why did you add  statics types in model? I thought we've got statusCode and strategyStateEnum already?